python/ml4ir/base/features/feature_config.py

```diff
@@ -265,41 +360,127 @@ def get_train_features(self, key: str = None):
         """
         Getter method for train_features in FeatureConfig object
         Can additionally be used to only fetch a particular value from the dict
+
+        Parameters
+        ----------
+        key : str, optional
+            Name of the configuration key to be fetched.
+            If None, then entire dictionary for the feature is returned
+
+        Returns
+        -------
+        list
+            Lift of feature configuration dictionaries or values for
+            trainable features in FeatureConfig
         """
         return self._get_list_of_keys_or_dicts(self.train_features, key=key)
 
     def get_metadata_features(self, key: str = None):
         """
         Getter method for metadata_features in FeatureConfig object
         Can additionally be used to only fetch a particular value from the dict
+
+        Parameters
+        ----------
+        key : str, optional
+            Name of the configuration key to be fetched.
+            If None, then entire dictionary for the feature is returned
+
+        Returns
+        -------
+        list
+            Lift of feature configuration dictionaries or values for
+            metadata features in FeatureConfig
         """
         return self._get_list_of_keys_or_dicts(self.metadata_features, key=key)
 
     def get_features_to_log(self, key: str = None):
         """
         Getter method for features_to_log in FeatureConfig object
         Can additionally be used to only fetch a particular value from the dict
+
+        Parameters
+        ----------
+        key : str, optional
+            Name of the configuration key to be fetched.
+            If None, then entire dictionary for the feature is returned
+
+        Returns
+        -------
+        list
+            Lift of feature configuration dictionaries or values for
+            features to be logged at inference
         """
         return self._get_list_of_keys_or_dicts(self.features_to_log, key=key)
 
     def get_group_metrics_keys(self, key: str = None):
         """
-        Getter method for group_metrics_keys in FeatureConfig object
+        Getter method for group_metric_keys in FeatureConfig object
```

Comment:
I am confused
is should be `group_metrics_keys`, right ?
like in line 433 ?

Reply:
Fixed.